### DISCUSSION TITLE
Shortcut issues with Morgan

### DISCUSSION THREAD

**Discussion by JackYansongLi** [MoganLab/mogan#244] [Q&A]
I just transfer from Xmacs to Morgan these days and there are quite a few problems to make me go back to Xmacs.

1. the copy and paste shortcut in Morgan is incredibly difficult to use, it uses ALT+W and CTRL+Y to copy and paste.
2. when I shift select a word "x" and type "y", it just added "x" before "y", that is, I want to replace "x" by "y", but it gives me "xy".

**Comment by YiqiXu** [comment]
Concerning the first point, @darcy-shen has set Emacs-style keyboard shortcuts as the default for consistency across various platforms. You could always change it to your preferred style through `Edit → Preferences... → General → Look and feel`.

**Comment by da-liii** [comment]
> when I shift select a word "x" and type "y", it just added "x" before "y", that is, I want to replace "x" by "y", but it gives me "xy".

I can reproduce this feature (or bug). I think it is by design. And of course, we can change the behavior for Mogan Editor to make it more consistent with normal text editor.

**Reply by YiqiXu** [reply]
Is it specific to Windows? I can't reproduce it on macOS.

**Reply by da-liii** [reply]
I reproduce it on macOS.
